src/components/views/read/cvForm/FormLayout.tsx

```diff
@@ -0,0 +1,327 @@
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is education
+              </p>
+            )}
+            {activeStep === 3 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is work experience
+              </p>
+            )}
+            {activeStep === 4 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is skills
+              </p>
+            )}
+            {activeStep === 5 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is portfolio
+              </p>
+            )}
+            {activeStep === 6 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is personal motivation
+              </p>
+            )}
+            {activeStep === 7 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is reference
+              </p>
+            )}
+            {activeStep === 8 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is template
+              </p>
+            )}
+            {activeStep === 9 && (
+              <p
+                style={{
+                  textAlign: 'center',
+                  alignSelf: 'center',
+                  justifyContent: 'cetner',
+                }}
+              >
+                hello, this is review
+              </p>
+            )}
+          </div>
+          <div
+            style={{
+              alignSelf: 'stretch',
+              position: 'relative',
+              height: '194px',
+              display: 'none',
+            }}
+          >
+            <div
+              style={{
+                position: 'absolute',
+                height: '100%',
+                width: '100%',
+                top: '0%',
+                right: '0%',
+                bottom: '0%',
+                left: '0%',
+              }}
+            />
```

Comment:
Why the self-closing div ? Please make sure you have eslint enabled and that these issues are showing in your IDE. To the very least they are showing in the terminal when you launch the app, and you should resolve them.

Reply:
It appeared that Eslint wasn't installed, so I just have installed it.